bigbluebutton-html5/imports/ui/components/audio/service.js

```diff
@@ -29,6 +30,16 @@ const init = (messages) => {
   AudioManager.init(userData);
 };
 
+const audioLocked = () => {
+  const userId = Auth.userID;
+  const User = mapUser(Users.findOne({ userId }));
+
+  const Meeting = Meetings.findOne({ meetingId: Auth.meetingID });
+  const audioLock = Meeting.lockSettingsProp.disableMic;
```

Comment:
![screenshot from 2018-07-11 16-02-10](https://user-images.githubusercontent.com/6312397/42596491-d3d2d7f8-8523-11e8-9a89-f6e6a6bba7fa.png)

Reply:
I hit this if I'm the first user in a meeting (i.e. I create the meeting as I join from HTML5). If I first join via Flash it's ok. Please add some check

Reply:
The lock settings are initialized from config.xml by the first moderator to join using the Flash client. What we do in the Flash client is have a set of predefined lock settings that are applied when a non-moderator joins and then are overwritten as necessary. You guys might want to do a similar thing with your HTML5 config.

Reply:
We should probably add [an empty] `lockSettingsProp` on meeting create - as @capilkey mentioned

Reply:
Not necessarily empty, but you could have a pre-filled lock settings so that administrators can set a default when they're using HTML5 only.